### ISSUE TITLE
Missing data due to Internet, System or other outage

### ISSUE THREAD

**Issue by gauthig** [jertel/vuegraf#132]
Based on discussion point 131, need a procedure to daily find missing data and pull it from emporia.

If you approve, please tag as enhancement and assign it to me. ETA 1 week.

**Comment by jertel** [comment]
Absolutely! This sounds like a very useful enhancement, as I've seen Emporia have a few short outages over the past couple of weeks.
